src/meltano/cli/install.py

```diff
@@ -8,6 +8,7 @@
 from meltano.core.project_plugins_service import ProjectPluginsService
 from meltano.core.tracking import CliEvent, PluginsTrackingContext
 
+from ..core.tracking.tracker import Tracker
```

Comment:
I know this technically works, just making sure it's intentional (ditto for the similar one below).

Reply:
Good catch. It must've been added automatically, since I certainly didn't write that. I'll remove it.

Reply:
The auto import hook from my editor used to do this occasionally - so if you come across some pre-existing ones, they're probably my fault 😁